data/news/platform/ocaml-4.14.1.rc1.md

```diff
@@ -0,0 +1,149 @@
+description: First Release Candidate of OCaml 4.14.1
+date: "2022-12-08"
+tags: [ocaml, platform]
+---
+
+The release of OCaml version 4.14.1 is imminent.
+
+This companion release to the OCaml 5.0.0 release will backport many safe bug
+fixes from the currently experimental 5.0 branch to the stable 4.14 branch.
+A full list of bug fixes is available below.
+
+In order to ensure that the future release works as expected, we are testing
+a release candidate during the upcoming weeks.
+
+If you find any bugs, please report them here on [GitHub](https://github.com/ocaml/ocaml/issues).
+
+----
+
+
+## Installation Instructions
+
+The base compiler can be installed as an opam switch with the following commands on opam 2.1:
+```
+opam update
+opam switch create 4.14.1~rc1
+```
+For previous version of opam, the switch creation command line is slightly more verbose:
+```
+opam update
+opam switch create 4.14.1~rc1 --repositories=default,beta=git+https://github.com/ocaml/ocaml-beta-repository.git
+```
+It might be also interesting to check the new support for parallelism by installing the `domainslib` library with
+```
+opam install domainslib
+```
+The source code for the release candidate is available on
+
+- [GitHub](https://github.com/ocaml/ocaml/archive/4.14.1-rc1.tar.gz)
+- [Inria archives](https://caml.inria.fr/pub/distrib/ocaml-4.14.1/ocaml-4.14.1~rc1.tar.gz)
+
+### Fine-Tuned Compiler Configuration
+
+If you want to tweak the configuration of the compiler, you can switch to the option variant with:
+```bash
+opam update
+opam switch create <switch_name> ocaml-variants.4.14.1~rc1+options <option_list>
+```
+where `<option_list>` is a comma-separated list of `ocaml-option-*` packages. For instance, for a `flambda` and `no-flat-float-array` switch:
+```bash
+opam switch create 4.14.1~rc1+flambda+nffa ocaml-variants.4.14.1~rc1+options ocaml-option-flambda ocaml-option-no-flat-float-array
+```
+The command line above is slightly more complicated for opam versions before 2.1:
+```bash
+opam update
+opam switch create <switch_name> --packages=ocaml-variants.4.14.1~rc1+options,<option_list> --repositories=default,beta=git+https://github.com/ocaml/ocaml-beta-repository.git
+```
+In both cases, all available options can be listed with `opam search ocaml-option`.
+
+---
+
+## Changes Since OCaml 4.14.0
+
+
+### Compiler User-Interface and Warnings:
+
+- [#11184](https://github.com/ocaml/ocaml/issues/11184), [#11670](https://github.com/ocaml/ocaml/issues/11670): Stop calling ranlib on created / installed libraries
+  (Sébastien Hinderer and Xavier Leroy, review by the same)
+
+### Build System:
+
+- [#11370](https://github.com/ocaml/ocaml/issues/11370), [#11373](https://github.com/ocaml/ocaml/issues/11373): Don't pass CFLAGS to flexlink during configure.
+  (David Allsopp, report by William Hu, review by Xavier Leroy and
+   Sébastien Hinderer)
+
+- [#11487](https://github.com/ocaml/ocaml/issues/11487): Thwart FMA test optimization during configure
```

Comment:
```suggestion
- [#11487](https://github.com/ocaml/ocaml/issues/11487): Thwart FMA test optimisation during configure
```
The Tarides Style Guide goes with the UK spelling, but I’m not sure that applies here.